frontend/src/pages/wifiPage/StandaloneWifiPageContainer.tsx

```diff
@@ -34,16 +39,48 @@ export default () => {
     fetchNetworks();
   }, []);
 
+  const waitForAdvancedConfigUrlTimeout = 700;
+
+  const waitForAdvancedConfigUrl = () => {
+    const interval = setInterval(async () => {
+      try {
+        getVncWpaGuiUrl()
+          .then((data: any) => {
+            if (data.url !== "") {
+              clearInterval(interval)
+              setAdvancedConfigUrl(data.url);
+            }
+          })
```

Comment:
we generally prefer to use async await instead of the promise .then callbacks etc.

also, try-catch only works with the await form, for the promise style you have to use .catch callback. as it is currently, if getVncWpaGuiUrl throws/rejects it won't be caught by your catch block, and will be logged to the browser console as an uncaught promise rejection.

```suggestion
    const interval = setInterval(async () => {
      try {
        const data = await getVncWpaGuiUrl();
        if (data.url !== "") {
          clearInterval(interval);
          setAdvancedConfigUrl(data.url);
        }
```